.github/CONTRIBUTING.md

```diff
@@ -37,6 +37,11 @@ Some hints:
  * Models sent/received by the controllers are defined in [/shared/models](/shared/models) directory
 
 
+## Improve the website
+
+PeerTube's website is [joinpeertube.org](https://joinpeertube.org), where people can learn about the project and how it works – not that it is not a PeerTube instance, but more the “marketing website”. You can help us improve it too! It is not hosted on GitHub but rather on [Framasoft](https://framasoft.org/)'s own [GitLab](https://about.gitlab.com/) instance, [FramaGit](https://framagit.org): https://framagit.org/framasoft/peertube/joinpeertube
```

Comment:
Should be "*note* that it's not a Peertube instance". Also you should add newlines.

Reply:
Oops, it was a typo 😊